agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -381,7 +371,7 @@ impl MpidMessage {
                                                      public_key) && self.header.verify(public_key)
     }
 
-    fn encode(recipient: &::NameType, body: &Vec<u8>) -> Vec<u8> {
+    fn encode(recipient: &XorName, body: &Vec<u8>) -> Vec<u8> {
         ::utils::encode(&(recipient, body)).unwrap_or(vec![])
```

Comment:
Is this encode/decode util stuff still available, or it gives way to the serialise/deserialise functions?
And, shall we prefer a wrapper class to holds the fields (recipient, body), so serialise/deserialise can be used smoothly.